invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -121,6 +140,10 @@ export const RecordVersionsList = ({ record, isPreview }) => {
         </Placeholder>
       </>
     )
+  ) : error ? (
```

Comment:
we have 3 level of nested ternary operator here, could it be solved in any other way instead to improve code readability ?